libs/design/scss/layout/_breakpoint.scss

```diff
@@ -1,15 +1,21 @@
 @use 'sass:map';
 @use 'variables' as v;
 
-//
-// Take a the list of breakpoints and retrieves the defined point.
-// If an invalid breakpoint is defined, it will display a warning.
-//
-// @usage
-// ```
-// @include layout.breakpoint(mobile) {}
-// ```
-//
+/// Applies styles at a specific breakpoint defined by `$breakpoints`.
+///
+/// This mixin looks up the breakpoint in the `$breakpoints` map and wraps
+/// the content in a min-width media query. If an invalid breakpoint is defined,
+/// it will display a warning.
+///
+/// @group Layout
+///
+/// @param {String} $point — The name of the breakpoint (e.g., `mobile`, `tablet`, `desktop`).
+///
+/// @example scss
+/// 	@include breakpoint(tablet) {
```

Comment:
Can you include a full example (including the imports) that a end-user would use?

Reply:
done